builtin/common/misc_helpers.lua

```diff
@@ -400,6 +400,80 @@ function core.explode_scrollbar_event(evt)
 	return retval
 end
 
+--------------------------------------------------------------------------------
```

Comment:
Do we really need this clutter? Feel free to insert some blank lines, but these separators are BS and add absolutely no value.

Reply:
This matches the rest of builtin, out-of-scope for this PR (but I agree, it should just be two blank lines)

Reply:
Yeah, just for consistency with builtin--I don't like it either.